Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -71,7 +64,7 @@ struct KalmanFitterOptions {
   KalmanFitterOptions(std::reference_wrapper<const GeometryContext> gctx,
                       std::reference_wrapper<const MagneticFieldContext> mctx,
                       std::reference_wrapper<const CalibrationContext> cctx,
-                      const OutlierFinder& outlierFinder_,
+                      Calibrator&& calibrator_, OutlierFinder&& outlierFinder_,
```

Comment:
Why not simply by value? They should be trivial structs anyway.

Reply:
Yes, they should be [semiregular](https://en.cppreference.com/w/cpp/concepts/semiregular) since we also copy them by value later on. I switched to by-value usage (also in the CKF options).